src/aggregator/integration/integration_data.go

```diff
@@ -781,8 +781,7 @@ func (mu metricUnion) ID() metricid.RawID {
 type metadataType int
 
 const (
-	policiesListType metadataType = iota
-	stagedMetadatasType
+	stagedMetadatasType metadataType = iota
```

Comment:
nit: just sanity checking that these values aren't used externally - otherwise we should `_` this first one to maintain the existing iota values